src/pool.ts

```diff
@@ -428,7 +440,7 @@ export class Pool {
    * - deposit of asset B
    * - "ADDLIQ" application call to add liquidity with the above deposits
    *
-   * If the pool is empty and the product of both assets is larger then 2**64 then an additional set of 3 transactions is built.
+   * For constant product pools only - if the pool is empty and the product of both assets is larger then 2**64 then an additional set of 3 transactions is built.
```

Comment:
`larger then 2**64` should be `larger or equal than 2**64` 

For the record with new constant product pools one can send only 2 transactions. We can handle product over 2^64-1